homeassistant/components/dialogflow/strings.json

```diff
@@ -8,7 +8,7 @@
     },
     "abort": {
       "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
-      "not_internet_accessible": "Your Home Assistant instance needs to be accessible from the internet to receive Dialogflow messages."
+      "webhook_not_internet_accessible": "[%key:common::config_flow::abort::not_internet_accessible%]"
```

Comment:
Please update the reference value too to use the correct common key.

Reply:
Forgot that :sweat_smile: changed now.